src/QuadForm/Herm/LocallyIsometricSublattice.jl

```diff
@@ -147,47 +151,44 @@ function _locally_isometric_sublattice_odd_ramified(M, L, p, P, absolute_map)
   if length(B0) == 0
     LL = lattice(ambient_space(M), _module_scale_ideal(P, pseudo_matrix(M)))
   else
-    _new_pmat = _sum_modules(pseudo_matrix(reduce(vcat, B0)), _module_scale_ideal(P, pseudo_matrix(M)))
+    Ppm = _module_scale_ideal(P, pseudo_matrix(M))
+    _new_pmat = _sum_modules(pseudo_matrix(reduce(vcat, B0)), Ppm)
     _new_pmat = _intersect_modules(_new_pmat, pseudo_matrix(M))
     LL = lattice(ambient_space(M), _new_pmat)
   end
-  @assert genus(LL, p) == c
 
   k, h = ResidueField(order(p), p)
   hext = extend(h, K)
   for j in length(C)-1:-1:1
     c = C[j]
     if all(!(scale(c, i) in [0, 1]) for i in 1:length(c))
-      @assert scale(C[1], 1) - valuation(scale(LL), P) >= 0
       s = div(scale(C[1], 1) - valuation(scale(LL), P), 2)
-      LL = lattice(ambient_space(LL), P^s * pseudo_matrix(LL))
+      LL = lattice(ambient_space(LL), _module_scale_ideal(P^s, pseudo_matrix(LL)))
       break
     end
     B, G, S = jordan_decomposition(LL, p)
     r = findfirst(i -> scale(c, i) == 1, 1:length(c))
     if r !== nothing
       r = rank(c, r)
       i = findfirst(j -> j == 1, S)
-      @assert i !== nothing
```

Comment:
Why?

Reply:
I made it run over a bunch of examples and it seems that the codes never had any problem of assertion so I assumed that these were debug lines, and I just deleted them to lighten the codes. But I could put them back is you think we should keep them :)

Reply:
Those are super cheap so I would be in favour of just keeping them.